keyboards/evyd13/atom47/keymaps/PHSC138/README.md

```diff
@@ -0,0 +1,71 @@
+# PHSC138's Layout for the atom47
```

Comment:
please rename this file and folder to be lower case.

Reply:
Resolved here [b1e0652](https://github.com/qmk/qmk_firmware/pull/18768/commits/b1e065224b86b31be3432a7808fbfa6a0d9f985a)